store/store.go

```diff
@@ -556,6 +557,19 @@ func (s *Store) refreshDeviceSession(device *auth.DeviceState) error {
 		return fmt.Errorf("internal error: no authContext")
 	}
 
+	s.sessionMu.Lock()
```

Comment:
Should this use "s.SessionLock()" (and same for the unlock below)? I guess it does not make any practical difference but maybe for consistency?

Reply:
those are just there for tests, they are defined in export_test.go